woocommerce/woocommerce
# Update onboarding task documentation & add back `example` command

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Related https://github.com/woocommerce/woocommerce/issues/33293

Update onboarding task documentation 

1. Add back `pnpm --filter=@woocommerce/admin-library example` command so that we can build the examples.
2. Update the example's webpack config to ignore the output directory to avoid infinite rebuilds
3. Improve onboarding task documentation
     - Add readme to add task example
     - Update `docs/extension-development/handling-merchant-onboarding.md` (It's outdated.)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Review the documentation changes.
2. Run `WC_EXT=add-task pnpm --filter=@woocommerce/admin-library example`
3. Confirm `plugins/add-task` is created.
4. Activate the plugin and confirm the task is added to the list.
5. Run `WC_EXT=add-task pnpm --filter=@woocommerce/admin-library example --watch`
6.  Confirm it watches for changes and only rebuilds when the file changes.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update onboarding task documentation and fix example command

#### Comment 

</details>
